config/default.yml

```diff
@@ -3550,6 +3550,7 @@ Style/For:
   SupportedStyles:
     - each
     - for
+  SafeAutoCorrect: false
```

Comment:
Can you move the `SafeAutoCorrect: false` under `Enabled: true` and update `VersionChanged`?

```diff
Style/For:
  Description: 'Checks use of for or each in multiline loops.'
  StyleGuide: '#no-for-loops'
  Enabled: true
+ SafeAutoCorrect: false
  VersionAdded: '0.13'
+ VersionChanged: '<<next>>'
  EnforcedStyle: each
  SupportedStyles:
    - each
    - for
```